docs/docs/configuration/integration.md

```diff
@@ -274,29 +274,41 @@ This example is for a simple reverse proxy setup where the `/oauth2/` path is ke
 **Following options need to be set on `oauth2-proxy`:**
 - `--reverse-proxy=true`: Enables the use of `X-Forwarded-*` headers to determine redirects correctly
 
-```nginx
-{{ domain }} {
-	# define forward auth for any path under `/`, if not more specific defined
-	forward_auth / {{ oauth.internalIP }}:4180 {
-		uri /oauth2/auth
-		copy_headers Authorization X-Auth-Request-User X-Auth-Request-Email
-
-		@error status 401
-		handle_response @error {
-			redir * /oauth2/sign_in?rd={scheme}://{host}{uri} 302
```

Comment:
Why did we remove the scheme and host from the redirect? It is preferential to use the full URL vs just the URI typically

Reply:
I changed it because the redirect didn't work in my setup, but actually it turned out that the problem was caused by missing `--whitelist-domain` in my configuration. Reverted in e38221bf693612fee6a4d2e19294248b1d02b64e.